MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+
+{% tabs %}
+{% highlight c# %}
+this.listView.AutoScroller.Interval = new TimeSpan(0, 0, 0, 0, 200);
+{% endhighlight %}
+{% endtabs %}
+
+### Disable outside scroll
+
+To disable auto-scroll when dragging item moves outside the `SfListView` while dragging, set the `AllowOutsideScroll` property of `AutoScroller` to `false`. The default value is `true`.
+
+{% tabs %}
+{% highlight c# %}
+this.listView.AutoScroller.AllowOutsideScroll = false;
+{% endhighlight %}
+{% endtabs %}
+
+## Disable dragging for particular item
+
+To disable dragging for a particular item, handle the `ItemDragging` event based on the conditions of `Action` event argument.
+
+You can cancel the dragging action for a particular item by setting the `Cancel` property of the `ItemDraggingEventArgs`.
+
+{% tabs %}
+{% highlight c# %}
+private void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+  // Disable the dragging for 4th item.
+  if (e.Action == DragAction.Start && e.NewIndex == 3)
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Cancel dropping for the dragged item
+
+To cancel dropping for the dragged item, handle the `ItemDragging` event based on the conditions of `Action` event argument. 
+
+You can cancel the dropping action for a particular item by setting the `Cancel` property of the `ItemDraggingEventArgs`.
+
+{% tabs %}
+{% highlight c# %}
+using Syncfusion.Maui.ListView.Helpers;
+private void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+  // Cancel the dropping if drop the drag item into out of view.
+  var listView = sender as ListView;
+  var totalExtent = this.listView.GetVisualContainer().Bounds.Bottom;
+  if (e.Action == DragAction.Drop && (e.Bounds.Y < -30 || e.Bounds.Bottom > totalExtent + 40))
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Reorder the underlying collection
+
+The underlying collection can be reordered directly by setting the `UpdateSource` property to `true`. The default value is `false`. 
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnHold"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60">
+  <syncfusion:SfListView.DragDropController>
+                <syncfusion:DragDropController
+                    UpdateSource="True">
+                </syncfusion:DragDropController>
+   </syncfusion:SfListView.DragDropController>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.listView.DragDropController.UpdateSource = true;
+{% endhighlight %}
+{% endtabs %}
+
+We can able to update collection even when `UpdateSource` is `false`. Like, user can decide where dragged item should be dropped actually by handling the `ItemDragging` event with `DragAction.Drop`.
```

Comment:
You can update the collection even when `UpdateSource` is `false`. Like, the user can decide where the dragged item should be dropped actually by handling the `ItemDragging` event with `DragAction.Drop`.

Reply:
Addressed the concerns provided.